src/components/admonition/admonition.tsx

```diff
@@ -0,0 +1,105 @@
+import { Component, h, Prop } from '@stencil/core';
+import { AdmonitionType } from '@limetech/lime-elements';
+
+/**
+ * Admonitions, also known as call-outs, very useful for including supportive content
+ * inside a user interface, or within a large piece of text.
+ *
+ * When used in a document or text based user interface, the admonitions attract
+ * reader's attention to a particular piece of information,
+ * without significantly interrupting their flow of reading the document.
+ *
+ * On a user interface, their usage will be experienced more intrusively by
+ * end-users. But they could be good choices when you indent to slightly
+ * disturb users' attention, and challenge them to pay extra attention
+ * to the information presented. In such cases, admonitions should not
+ * be used as static and constantly present elements of a UI. They should
+ * rather be displayed when something unusual or remarkable demands
+ * users attention.
+ *
+ * @exampleComponent limel-example-admonition-info
+ * @exampleComponent limel-example-admonition-warning
+ * @exampleComponent limel-example-admonition-danger
+ * @exampleComponent limel-example-admonition-note
+ * @exampleComponent limel-example-admonition-tip
+ * @exampleComponent limel-example-admonition-custom-title-description
+ * @exampleComponent limel-example-admonition-composite
+ */
+@Component({
+    tag: 'limel-admonition',
+    shadow: true,
+    styleUrl: 'admonition.scss',
+})
+export class Admonition {
+    /**
+     * Title of the admonition
+     */
+    @Prop({ reflect: true })
+    public title?: string;
```

Comment:
<img width="539" alt="image" src="https://user-images.githubusercontent.com/35954987/227201650-568dd7ac-b03b-43ff-a786-268b97240b7a.png">

I'll change it to `heading`. It makes more sense too